src/components/StroyDetail.js

```diff
@@ -0,0 +1,188 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import styled from "styled-components";
+import axios from "axios";
+
+import Modal from "./common/Modal";
+import SubHeader from "./common/SubHeader";
+import StyledLoadingSpinner from "./shared/StyledLoadingSpinner";
+import {
+  STORY_SUB_HEADER_HEADING,
+  STORY_SUB_HEADER_PARAGRAPH,
+} from "../constants/story";
+
+export default function StoryDetail() {
+  const { counsel_id } = useParams();
+
+  const [story, setStory] = useState(null);
+  const [errorMessage, setErrorMessage] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isModalOn, setIsModalOn] = useState(false);
+
+  useEffect(() => {
+    (async () => {
+      const { data } = await axios.get(
+        `${process.env.REACT_APP_LOCAL_SERVER_URL}/api/counsels/${counsel_id}`,
+        {
+          withCredentials: true,
+        }
+      );
+
+      if (data.message) {
+        setErrorMessage(data.message);
+        setIsModalOn(true);
+        setIsLoading(false);
+
+        return;
+      }
+
+      setStory(data.data);
+      setIsLoading(false);
+    })();
+  }, [setStory]);
+
+  return (
+    <>
+      <SubHeader
+        heading={STORY_SUB_HEADER_HEADING}
+        paragraph={STORY_SUB_HEADER_PARAGRAPH}
+      />
+      <MainContainer>
+        {isLoading && <StyledLoadingSpinner />}
+        {!isLoading && errorMessage && isModalOn ? (
+          <Modal onClick={setIsModalOn} width="150px" height="50px">
+            {errorMessage}
+          </Modal>
+        ) : null}
+        {!isLoading && !errorMessage && story ? (
+          <>
+            <section>
+              <StoryHeaderWrapper>
+                <ImageWrapper>
+                  <img src={story.counslee.imageURL} alt="Profile Image" />
+                </ImageWrapper>
+                <StoryInfoWrapper>
+                  <div className="name">{story.counselee.nickname}</div>
```

Comment:
```suggestion
                  <div className="name">{story. counslee.nickname}</div>
```
제가 만든 목데이터에 오타나게 적어나서 똑같이 쓰신 것 같아요 ㅜㅜ 바꿔주세용!

Reply:
백엔드 확인해보니 알맞게 작성 하신 것 같습니다!